tests/tuning/hyperparams/test_boolean.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+from unittest import TestCase
+
+from btb.tuning.hyperparams.boolean import BooleanHyperParam
+
+
+class TestBooleanHyperParam(TestCase):
+    """Unit test for the class ``BooleanHyperParam``."""
+
+    def setUp(self):
+        self.instance = BooleanHyperParam()
+
+    def test_trasnform_list(self):
+        """Test that the method ``transform`` performs a normalization over a ``list`` of boolean
+        values and converts them in to a search space of [0, 1]^k.
+        """
+
+        # setup
+        values_1 = [True, False, True]
+        values_2 = [False, False, True]
```

Comment:
i don't understand why two separate values are needed here. is there something interesting about the second set of values? all you are testing is that they are transformed into integers.